zerver/migrations/0698_namedusergroup_rendered_description.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.0.10 on 2025-01-15 20:05
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0697_empty_topic_name_for_dms_from_third_party_imports"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="namedusergroup",
+            name="rendered_description",
+            field=models.TextField(default=""),
```

Comment:
Provide a `db_default` for this -- otherwise, it'll be an expensive migration to apply in production.

Also, is `""` what you get when running the Markdown processor on the empty string? We likely want to match that.